vpc.tf

```diff
@@ -0,0 +1,17 @@
+module "vpc" {
+    source = "github.com/terraform-aws-modules/terraform-aws-vpc"
+    
+    name = "test-vpc"
+    cidr = "10.1.0.0/16"
+    azs  = ["eu-west-1a", "eu-west-1b"]
+    private_subnets = ["10.1.0.0/18", "10.1.128.0/18"]
+    public_subnets  = ["10.1.64.0/18", "10.1.192.0/18"]
```

Comment:
all cidr values shall be moved into locals { } section, refactor plz

Reply:
Explain, please